src/sysv/systemd/sssd-sudo.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=SSSD Sudo Service responder
+Documentation=man:sssd.conf(5)
```

Comment:
Two questions about documentation:
1) shouldn't we also point to `man sssd-sudo`?
2) shouldn't we mention that the sudo socket can be enabled instead of configuring the service in `man sssd-sudo`?